client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -137,12 +137,12 @@ class SharingServiceDescription extends Component {
 			},
 			slack: function () {
 				if ( this.props.numberOfConnections > 0 ) {
-					return this.props.translate( 'Connected this P2 to your Slack Workspace.', {
+					return this.props.translate( 'Workspace connected to Slack.', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.